Bio/SeqUtils/MeltingTemp.py

```diff
@@ -527,10 +565,16 @@ def salt_correction(Na=0, K=0, Tris=0, Mg=0, dNTPs=0, method=1, seq=None):
         corr = 11.7 * math.log10(mon)
     if method == 5:
         corr = 0.368 * (len(seq) - 1) * math.log(mon)
+    # Turn black code style off
+    # fmt: off
     if method == 6:
         corr = (4.29 * SeqUtils.GC(seq) / 100 - 3.95) * 1e-5 * math.log(mon) +\
             9.40e-6 * math.log(mon) ** 2
+    # Turn black code style on
+    # fmt: on
     if method == 7:
```

Comment:
Wouldn't it be less noisy to let black turned off until line 603?

Reply:
Yes, updated now.